modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldValueRendererAccessorTest.java

```diff
@@ -54,46 +54,15 @@ public void testRenderMultipleValues() throws Exception {
 
 		int numberOfOptions = 2;
 
-		ddmFormField.setDDMFormFieldOptions(
-			createDDMFormFieldOptions(numberOfOptions));
-
-		ddmForm.addDDMFormField(ddmFormField);
-
-		DDMFormValues ddmFormValues = DDMFormValuesTestUtil.createDDMFormValues(
-			ddmForm);
-
-		JSONArray optionsValuesJSONArray = _createOptionsValuesJSONArray(
-			numberOfOptions);
-
-		DDMFormFieldValue ddmFormFieldValue =
-			DDMFormValuesTestUtil.createDDMFormFieldValue(
-				"Select",
-				new UnlocalizedValue(optionsValuesJSONArray.toString()));
-
-		ddmFormValues.addDDMFormFieldValue(ddmFormFieldValue);
-
-		SelectDDMFormFieldValueRenderer selectDDMFormFieldValueRenderer =
-			_createSelectDDMFormFieldValueRenderer();
-
-		Assert.assertEquals(
-			"option 1, option 2",
-			selectDDMFormFieldValueRenderer.render(
-				ddmFormFieldValue, LocaleUtil.US));
-	}
-
-	@Test
-	public void testRenderSingleValue() throws Exception {
-		DDMForm ddmForm = DDMFormTestUtil.createDDMForm();
-
-		DDMFormField ddmFormField = DDMFormTestUtil.createDDMFormField(
-			"Select", "Select", "select", "string", false, false, false);
+		DDMFormFieldOptions ddmFormFieldOptions = new DDMFormFieldOptions();
 
-		ddmFormField.setProperty("dataSourceType", "manual");
+		ddmFormFieldOptions.addOptionLabel(
+			"value 1", LocaleUtil.US, "option 1");
 
-		int numberOfOptions = 1;
+		ddmFormFieldOptions.addOptionLabel(
+			"value 2", LocaleUtil.US, "option with &");
 
```

Comment:
Hi @leticia-maciel, (**same as above**) can we keep the
```
ddmFormFieldOptions.addOptionLabel("value 2", LocaleUtil.US, "option 2");`
```
and add a third option
 
```
ddmFormFieldOptions.addOptionLabel("value 3", LocaleUtil.US, "option with &");
```
?